layouts/partials/styles.html

```diff
@@ -11,6 +11,11 @@
 <link href="{{ $css7.RelPermalink }}" integrity="{{ $css7.Data.Integrity }}" rel="stylesheet" type="text/css" id="css7"
   disabled>
 
+<link rel="preconnect" href="https://rsms.me/">
+<link rel="stylesheet" href="https://rsms.me/inter/inter.css">
```

Comment:
Do we need permission or some type of review to be able to use this project?

Our best practices are to download from source, include a copy in the repo, and load it using the hugo data integrity functions as above. That helps safeguard against risks to the source being compromised. One that's done, I'm happy to merge this in.

Reply:
It's fine licence wise, it's the standard OFL https://fonts.google.com/specimen/Inter/license.
I'll pull it directly into the repo alright, that's a good shout 👍